### ISSUE TITLE
MSI installer installs different dotnet cli versions on different versions of Windows

### ISSUE THREAD

**Issue by dsparkplug** [dotnet/cli#1672]
## Steps to reproduce

Download the MSI installer from https://dotnet.github.io/getting-started/ 
Install on Windows 10 Pro
Check version
```
 > dotnet --version
.NET Command Line Tools (1.0.0)
Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.10586
 OS Platform: Windows
 Runtime Id:  win10-x64 
```

Install on Windows Server 2012 Standard
Check version

```
 > dotnet --version
.NET Command Line Tools (1.0.0-beta-001598)

Product Information:
 Version:     1.0.0-beta-001598
 Commit Sha:  7582649f88

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.2.9200
 OS Platform: Windows
 Runtime Id:  win8-x64
```

## Expected  behavior

The same (non-beta) version installed on each operating system 

## Actual behavior

"stable" version installed on Windows 10 and beta version on Windows Server 2012

The main issue here is that the command line options (for dotnet build) are significantly different between versions. We were planning to provide support for this action in our CI server but cannot be sure which command line options to support. Even the commands are changing e.g. compile is gone in the later version

**Comment by blackdwarf** [comment]
@dsparkplug this is very weird. So you're saying you downloaded the same EXE, installed it on two different systems and it showed different versions? Did you by any chance have a previous CLI install (an older one) on the Win10 machine?

/cc @Sridhar-MS

**Comment by dsparkplug** [comment]
A previous version was already installed yesterday when I ran the installer on the Win10 machine.

Just uninstalled and then downloaded the installer (dotnet-win-x64.1.0.0.001598.exe) on Win10 machine again. Windows Defender detected it as Trojan:Win32/Gatrid.E!plock and quarantined it. 

I removed it from quarantine and scanned it on virustotal.com. It passed all scans, so I ran it again. It was then blocked by Windows Defender again (this time as !#UACTriger.A ). 

Did not attempt to run this any further.

It is possible that the dotnet-win-x64.1.0.0.001598.exe was quarantined without me noticing yesterday and I ran dotnet-win-x64.latest.msi, which exists in my downloads folder, instead. 

Therefore this seems to be an issue with Windows Defender instead of the installer. Still, probably best that you know about it.


.
